### PR TITLE
J-175: + min shownote time to 'timestamp' field in form

### PR DESCRIPTION
**Pull request by Denidzio**

### CODE REVIEW

File: frontend/src/components/configurate-episode/components/create-episode-form/components/shownote-input/shownote-input.tsx [hunk 1]
```diff
@@ -39,6 +40,7 @@ const ShownoteInput: React.FC<Props> = ({
           }
           control={control}
           errors={errors}
+          options={{ min: MIN_SHOWNOTE_TIME }}
```
Comment (what1s1ove) [inline frontend/src/components/configurate-episode/components/create-episode-form/components/shownote-input/shownote-input.tsx]:
```suggestion
```

File: frontend/src/components/common/input/input.tsx [hunk 2]
```diff
@@ -18,6 +18,8 @@ type Props = {
   control: Control;
   errors: FieldErrors;
   hasMultipleRows?: boolean;
+  options?: React.TextareaHTMLAttributes<HTMLTextAreaElement>
+  | React.InputHTMLAttributes<HTMLInputElement>;
```
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx]:
```suggestion
```

File: frontend/src/components/common/input/input.tsx [hunk 3]
```diff
@@ -37,12 +40,14 @@ const Input: React.FC<Props> = ({
...
         <textarea
           {...field}
+          {...(options as React.TextareaHTMLAttributes<HTMLTextAreaElement>)}
           placeholder={placeholder}
           className={getAllowedClasses(styles.input, styles.textarea)}
         />
       ) : (
         <input
           {...field}
+          {...(options as React.InputHTMLAttributes<HTMLInputElement>)}
```
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx:50]:
```suggestion
          min={MIN_SHOWNOTE_TIME}
```
what about if I will try to do it with the postman? You also need to change validation

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/input/input.tsx [hunk 4]
```diff
@@ -37,12 +40,14 @@ const Input: React.FC<Props> = ({
       {hasMultipleRows ? (
         <textarea
           {...field}
+          {...(options as React.TextareaHTMLAttributes<HTMLTextAreaElement>)}
```
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx:43]:
```suggestion
                    min={MIN_SHOWNOTE_TIME}
```

File: frontend/src/components/common/input/input.tsx (hunk 3 above)
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx:50]:
```suggestion
          min={MIN_SHOWNOTE_TIME}
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/input/input.tsx (hunk 3 above)
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx]:
```suggestion
          min={min}
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by JuliaDzyuba — DISMISSED** [review]

File: frontend/src/components/common/input/input.tsx [hunk 5]
```diff
@@ -18,7 +18,10 @@ type Props = {
   control: Control;
   errors: FieldErrors;
   hasMultipleRows?: boolean;
-};
+} & (
+  | React.InputHTMLAttributes<HTMLInputElement>
+  | React.TextareaHTMLAttributes<HTMLTextAreaElement>
+);
```
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx]:
```suggestion
}```
add prop directly

File: frontend/src/components/common/input/input.tsx [hunk 6]
```diff
@@ -28,6 +31,7 @@ const Input: React.FC<Props> = ({
   placeholder = '',
   type = InputType.TEXT,
   hasMultipleRows = false,
+  ...options
```
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx]:
```suggestion
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by EugeniusGR — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**